Tasks/PublishSymbolsV2/task.json

```diff
@@ -41,14 +41,14 @@
             "required": true,
             "helpMarkDown": "The pattern used to discover the pdb files to publish."
         },
-        {
```

Comment:
There appear to be some whitespace differences, so you probably want to make sure the file is indented correctly and that everything is either tabs or spaces